.github/workflows/latest-release.yml

```diff
@@ -3,27 +3,36 @@ on:
     # Sequence of patterns matched against refs/tags
     branches:
       - 'main'
+      - 'v*'
+
     paths: 
       - 'KubeArmor/**'
       - 'protobuf/**'
 
-name: Latest version release after testing KubeArmor image
+name: Create KubeArmor releases after testing the image
 
 jobs:
   build:
-    name: Create KubeArmor Release - 18.04
+    name: Create KubeArmor Release
     runs-on: ubuntu-18.04
     timeout-minutes: 20
     steps:
       - name: Checkout KubeArmor code
         uses: actions/checkout@v2
 
-      - name: Generate KubeArmor artifacts
-        run: GITHUB_SHA=$GITHUB_SHA ./KubeArmor/build/build_kubearmor.sh
-
       - name: Setup Enviroment
-        run: |        
+        id: vars
+        run: |
+          if [[ github.ref == 'refs/heads/main' ]]; then
+            echo ::set-output name=tag::stable
```

Comment:
shouldn't this be `latest` in place of `stable`?

Reply:
Yes, you are right, will change this.